templates/organization/edit-basics.html

```diff
@@ -162,6 +162,30 @@ <h2>{% trans "Edit basics" %}</h2>
             <input type="hidden" value="{{ form.instance.firstciteddate.get_value.confidence }}" name="firstciteddate_confidence">
         </div>
     </div>
+    <div class="row realstart-row field-bg">
```

Comment:
This works great for the `edit-basics.html` template! Unfortunately, we'll have to propagate this change to the `create-basics.html` template as well, thanks to #597 😅